### ISSUE TITLE
Secure clear ACL on HDC by validating the signature

### ISSUE THREAD

**Issue by billettc** [Hivemapper/odc-api#48]

**Comment by jayalberts** [comment]
Was this included in 5.1.6 release?

**Comment by Eduard-Voiculescu** [comment]
Yep, there is a new endpoint added for clear which needs a signature. Previous endpoint is also maintained until the react-native-app is updated with respective changes.

**Comment by jayalberts** [comment]
1.2.1 firmware failed test due to Wi-Fi Direct - expecting new build tomorrow
